lms/djangoapps/instructor/views/api.py

```diff
@@ -1023,22 +1023,26 @@ def get_problem_responses(request, course_id):
     Responds with BadRequest if problem location is faulty.
     """
     course_key = CourseKey.from_string(course_id)
-    problem_location = request.POST.get('problem_location', '')
+    # A comma-separated list of problem locations
+    problem_locations = request.POST.get('problem_location', '')
```

Comment:
Thank you for keeping backwards compatibility here. Could you also please add a comment to that effect? It might confuse folks that the param is singular but the var is plural.